pytest failure

Following PR #105. Failure with error

```
            # Basal Drag
>           - inner(Phi, (1.0 - fl_ex) * B2 * as_vector([u, v]))   # B2 spans 1e-7, 1e7,  2nd step u ~ 1e-18 to 1e2
            * dx
    
            # Driving Stress
            + ( div(Phi)*F - inner(grad(bed), W*Phi) )
            * dx
        )
E       TypeError: unsupported operand type(s) for *: 'Sum' and 'NoneType'

[...]/fenics_ice/fenics_ice/solver.py:407: TypeError
```

--- a/fenics_ice/solver.py
+++ b/fenics_ice/solver.py
@@ -517,7 +517,7 @@ class ssa_solver:
             else:
                 raise RuntimeError(f"Invalid sliding law: '{sl:s}'")
 
-            return B2
+        return B2
 
     def solve_mom_eq(self, annotate_flag=None):
         """Solve the momentum equation defined in def_mom_eq"""

--- a/fenics_ice/solver.py
+++ b/fenics_ice/solver.py
@@ -516,6 +516,8 @@ class ssa_solver:
             
             else:
                 raise RuntimeError(f"Invalid sliding law: '{sl:s}'")
+        else:
+            raise RuntimeError(f"Invalid sliding law: '{sl:s}'")
 
         return B2
 

